src/modules/reservation/reservation.js

```diff
@@ -0,0 +1,145 @@
+import { getSeaFoodItem, reservationsUrl } from '../getApi';
+import reservationCounter from './reserveCounter';
+import addReservation from './addReservation';
+
+const reservation = async (idMeal) => {
+  try {
+    const reservation = document.createElement('div');
+    reservation.id = 'popup';
+    reservation.classList = 'popup';
+    document.body.appendChild(reservation);
+
+    // popup window
+    const popupWindow = document.createElement('div');
+    popupWindow.classList = 'window';
+    reservation.appendChild(popupWindow);
+
+    const popupHead = document.createElement('div');
+    popupHead.classList = 'popup-head';
+    popupHead.id = 'popup-head';
+
+    popupWindow.appendChild(popupHead);
+
+    const xButton = document.createElement('p');
+    xButton.id = 'close';
+    xButton.classList = 'close';
+    xButton.innerHTML = 'X';
+    popupHead.appendChild(xButton);
+
+    const imagePopup = document.createElement('img');
+    const meal = await fetch(getSeaFoodItem + idMeal);
+    const { meals } = await meal.json();
+    const data = meals.find((card) => card.idMeal === idMeal);
+    const {
+      strMeal, strCategory, strArea, strMealThumb, strYoutube,
+    } = data;
+    imagePopup.src = strMealThumb;
+    imagePopup.classList = 'image-popup';
+    imagePopup.id = 'image-popup';
+    popupWindow.appendChild(imagePopup);
+
+    const popupHeading = document.createElement('h2');
+    popupHeading.id = 'title';
+    popupHeading.classList = 'title';
+    popupHeading.innerHTML = strMeal;
+    popupWindow.appendChild(popupHeading);
+
+    const detailsPopup = document.createElement('div');
+    detailsPopup.classList = 'details';
+    detailsPopup.id = 'details';
+    detailsPopup.innerHTML = ` 
+          <p class="meal" id="meal">Meal: ${strMeal}</p>
+          <p class="meal" id="meal">Category: ${strCategory}</p>
+          <p class="meal" id="meal">Area: ${strArea}</p>
+          <p class="meal" id="meal"><a href="${strYoutube}">How to make it!</a></p>
+          `;
```

Comment:
- [ ] Kindly ensure that you improve the size of the design of the `Rservation Popup` in order to looks smarter and sticks as much as possible to the given **[wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/images/Reservations.png)**(e.g.: **Espacially the size of the popup,...**)

Reply:
Added changes requested